PythonScript/src/Enums.h

```diff
@@ -209,175 +209,24 @@ enum VkKeys
 //lint +e849
 
 /* ++Autogenerated ---------------------------------------------- */
-enum CaseInsensitiveBehaviour

```

Comment:
@ClaudiaFrank Do you an idea why this is sorted differently now? Is this expected?

Reply:
Yes, this is expected because the createwrapper script
loops over a dictionary, which contrary to lists, do not
keep the ordering of its items.

I compared the old and new results and both are the same.